src/k8s-engine/k8s.ts

```diff
@@ -97,6 +90,11 @@ interface KubernetesBackendEvents {
    * Show a notification to the user.
    */
   'show-notification': (options: Electron.NotificationConstructorOptions) => void;
+
+  /**
+   * Emitted when the checkForExistingKimBuilder setting pref changes
+   */
+  'kim-builder-check-changed': (status: boolean) => void;
```

Comment:
We only ever call this with `status = false`; maybe call it `kim-builder-uninstalled` (and not take any parameters) to better match where it's emitted?

Reply:
Makes sense.